src/reducerMaps/tags.js

```diff
@@ -85,15 +85,23 @@ export default new Map([
   ],
   [ActionTypes.DELETE_TAG, isFetching],
   [ActionTypes.TAG_DELETED, (state, action) =>
-    update(state, {
+  {
+    const tagIndex = getTagIds({ tags: state }).indexOf(action.id);
+    let items = {};
+    if (tagIndex !== -1) {
+      items = {
```

Comment:
instead of `let items = {}`, move `newState` up and make this line `newState.items = {`